### DISCUSSION TITLE
Feedback on loading (like a loading percent)

### DISCUSSION THREAD

**Discussion by iTrooz** [Docile-Alligator/Infinity-For-Reddit#977] [Ideas]
I think it would be nice to have some kind of feedback on image/video loading, like "5Mio/6Mio downloaded" or "85% loaded"

I may try to submit a PR for this, but as of now this is only a feature request.

What do you think ?

**Comment by Wladefant** [comment]
@iTrooz The question here is always if the api can give us information about it.
Can you find something?

**Comment by iTrooz** [comment]
Hum.. I'm not sure this one depends on the API
I was thinking more of a "downloaded bytes %" like we have on browsers. We wouldn't need any info from the Reddit API, since we would get this information directly from the HTTP client "what has been downloaded/what is the total file size ?"

**Comment by Wladefant** [comment]
> Hum.. I'm not sure this one depends on the API
> I was thinking more of a "downloaded bytes %" like we have on browsers. We wouldn't need any info from the Reddit API, since we would get this information directly from the HTTP client "what has been downloaded/what is the total file size ?"

😅 sorry I thought the wrong thing, I thought you mean how mich people downloaded it haha

**Comment by Wladefant** [comment]
> I think it would be nice to have some kind of feedback on image/video loading, like "5Mio/6Mio downloaded" or "85% loaded"
> 
> I may try to submit a PR for this, but as of now this is only a feature request.
> 
> What do you think ?

@iTrooz I dont know if it is needed because no reddit normaly the videos are really small and therefore fast downloadable. This would just use extra power of the phone without a real need

**Comment by iTrooz** [comment]
> therefore fast downloadable.

I might be the only one to experience this, but *several times a day* I need to keep scrolling, or make a bet on when the image/video/gif will finally load, because it didn't load after 10 seconds and I don't know if it's close to finishing or if I need to wait a whole minute

At least an option turned off by default would be a huge help

**Comment by Wladefant** [comment]
> > therefore fast downloadable.
> 
> I might be the only one to experience this, but _several times a day_ I need to keep scrolling, or make a bet on when the image/video/gif will finally load, because it didn't load after 10 seconds and I don't know if it's close to finishing or if I need to wait a whole minute
> 
> At least an option turned off by default would be a huge help

Oh ok, then it would make sense for some people
